tools/NuGet/DynamoVisualProgramming.Core/Core.nuspec

```diff
@@ -13,14 +13,18 @@
 		2)ProtoCore.dll
 		3)DynamoShapeManager.dll 
 		4)DynamoUtilities.dll
+		5)Analysis.dll
+		6)DSCoreNodes.dll
+		7)DynamoInstallDetective.dll
```

Comment:
let's create a new package called NodeLibrary and include Analysis.dll, DSCoreNodes.dll, DSIronPython.dll to that package. DynamoInstallDetective needs to be part of Core package, as it is required for application integration.